docs/team/Sheikh-Umar.adoc

```diff
@@ -0,0 +1,58 @@
+= Sheikh Umar - Project Portfolio
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== PROJECT: CRM Book
+
+---
+
+== Overview
+
+CRM Book is a desktop application targeted for salespeople. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+This application allows salespeople to organize their Leads, Contacts, and Accounts that suits their liking, as well as address problems that symptomatic in the sales industry,
+such as identifying duplicates.
+
+== Summary of contributions
+
+* *Major enhancement*: Added the ability to retrieve a desired list of non-duplicate data.
+** What it does: Informs the user if a Lead to be added in is already in the CRM Book either as a Lead or a Contact, and allows the user to retrieve a list of either Leads or Contacts.
+** Justification: Salespeople often face the issue of duplicate data as the CRM applications that they are using does not detect duplicates. Furthermore, the CRM application that they are using may not be able to display Leads and Contacts separately.
+Hence, this enhancement tackles this issues so that the user will have a more pleasant experience with a CRM application.
+** Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
+
+* *Minor enhancement*: Created an alias command for basic commands so that the user can specify the desired action quickly.
+
+* *Code contributed*: [https://github.com/CS2103JAN2018-F11-B1/main/blob/master/collated/functional/Sheikh-Umar.md[Functional code]] [https://github.com/CS2103JAN2018-F11-B1/main/blob/master/collated/test/Sheikh-Umar.md[Test code]]
+
+* *Other contributions*:
+
+** Project management:
+*** Managed the collation of code of my team for releases `v1.4` - `v1.5rc` (2 releases) on GitHub
+** Documentation:
+*** Did cosmetic tweaks to existing contents of the User Guide: https://github.com[#14]
+** Community:
+*** PRs reviewed (with non-trivial review comments): https://github.com/CS2103JAN2018-F11-B1/main/pull/158[#158], https://github.com/CS2103JAN2018-F11-B1/main/pull/159[#159]
```

Comment:
You didn't even leave a comment when reviewing these

Reply:
I thought approving the PR was enough. Will make necessary changes.